integration_test/dapp_tests/configs/nftConfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "marketplace": "0x305E8dc769aae003c17fB08b9cc9108Cb4630E79",
```

Comment:
are these constants or different for each chain?

Reply:
These are for the arctic 1, and already deployed. I didnt create another config for atlantic-2, on the basis that they should be identical but I can deploy and create another config for atlantic 2 as well.

Reply:
gotcha, just wanted to make sure it would be identical on `atlantic-2`